src/abstractions/serialization/KiotaJsonSerializer.Deserialization.NonGeneric.cs

```diff
@@ -0,0 +1,105 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.\
+// ------------------------------------------------------------------------------
+
+#if NET5_0_OR_GREATER
+
+using System.Threading.Tasks;
+using System.Threading;
+using System.IO;
+using System;
+using System.Collections.Concurrent;
+using System.Diagnostics.CodeAnalysis;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Kiota.Abstractions.Serialization;
+public static partial class KiotaJsonSerializer
+{
+#if NET8_0_OR_GREATER
+#else
+    /// <summary>
+    /// Palceholder for .NET &lt; 8
+    /// </summary>
+    private class RequiresDynamicCodeAttribute : Attribute
```

Comment:
Here is how we've solved the problem across the board.
https://github.com/microsoft/kiota-dotnet/blob/650812d9b598f192e6e55424e327d2bcee49c99a/src/abstractions/RequestInformation.cs#L51
Can you align with this please?

Reply:
done

Reply:
Well it looks kind of ugly now ... but can't help that, I guess.

Reply:
yes it does. That's ok, consistency over "prettyness". Once we address this, we'll be able to do so across the libraries.